src/main/resources/templates/db_scripts/schema.json

```diff
@@ -83,6 +83,32 @@
       "fromModuleVersion": 6.1,
       "withMetadata": true,
       "populateJsonWithId": true
+    },
+    {
+      "tableName": "acquisitions_unit_assignments",
+      "fromModuleVersion": 6.1,
+      "withMetadata": true,
+      "populateJsonWithId": true,
```

Comment:
`populateJsonWithId` should not be used anymore because [PR #131](https://github.com/folio-org/mod-orders-storage/pull/131/files#diff-162ca424ba9ad000edded275876b325b) migrated to RMB `25.0.1`